src/clipboard.js

```diff
@@ -55,16 +58,17 @@ export function paste (element, cursor, clipboardContent) {
  * @param {DOM node} A container where the pasted content is located.
  * @returns {Array of Strings} An array of cleaned innerHTML like strings.
  */
-export function parseContent (element) {
+export function parseContent (element, plainText = false) {
   // Filter pasted content
-  return filterHtmlElements(element)
+  return filterHtmlElements(element, plainText)
```

Comment:
The pasting with plainText should also have a few tests.

I suspect you filter the tags out too early and the splitting into blocks doesn't work anymore. 

And instead of using `content.getPlainText(elem)` I think you should be able to just use an empty array for `allowedElements`.